### PR TITLE
fix: variables corrrection in cart and payment router; readability

### PR DESCRIPTION
**Pull request by seyiadel**

### CODE REVIEW

File: cart/router.py [hunk 1]
```diff
@@ -13,65 +13,56 @@
...
-    "/add_to_cart",
-)
-async def add_to_cart(
-    item: List[schemas.CartItem],
-    user: dict = Depends(get_current_user),
-    db: Session = Depends(get_db),
-):
+@router.post("/add_to_cart")
+
+async def add_to_cart(items: List[schemas.CartItem], user: dict = Depends(get_current_user),db: Session = Depends(get_db)):
```
Comment (madvirus-ops) [inline cart/router.py:18]: Wetin come be the difference?

**Review by madvirus-ops — COMMENTED** [review]
Eh?

File: cart/router.py (hunk 1 above)
Comment (seyiadel) [reply cart/router.py:18]: The arrangement; each parameter was on a line

Comment (madvirus-ops) [reply cart/router.py:18]: Na formatting bro
You no dey fornat your code?

**Pull request merged**